resources/views/user/index.blade.php

```diff
@@ -22,6 +22,9 @@
                     @if(\LibreNMS\Authentication\LegacyAuth::getType() == 'mysql')
                     <th data-column-id="enabled" data-formatter="enabled">@lang('Enabled')</th>
                     @endif
+                    @if(\LibreNMS\Config::get('twofactor'))
```

Comment:
You can just use `@config('twofactor')`  and `@endconfig` (They are custom directives LibreNMS has)